### DISCUSSION TITLE
Make the wrench more compatible by using the c:wrenches tag (to rule them all)

### DISCUSSION THREAD

**Discussion by FederAndInk** [TechReborn/TechReborn#2936] [Ideas]
MI and AE are already using this tag for their wrenches, maybe others too.
To the unification of the wrenches! One wrench to rule them all! :)

**Comment by Ayutac** [comment]
the tag itself isn't the problem, but that wouldn't automatically transfer the functionality, as these are item-class-bound.

@Technici4n how does AE do it?

**Reply by Technici4n** [reply]
https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/master/src/main/java/appeng/hooks/WrenchHook.java

**Reply by Ayutac** [reply]
Thanks, looks good!
